### ISSUE TITLE
What does "dont Unpack" mean

### ISSUE THREAD

**Issue by ttodua** [ExeinfoASL/ASL#1]
is there a bug or what does it mean to show "don't" and then "Unpack" (with uppercase)- so, the meaning is vague. should we unpack or dont unpack?

https://i.imgur.com/y51Ye6P.png

**Comment by ExeinfoASL** [comment]
I tested de4dot and unable to unpack this protection.
Info is for you , don't try you wasting your time , look for another method

**Comment by ttodua** [comment]
yes, just the message should be a bit clear, like : "don't waste your time unpacking with de4dot".... thanks
